### DISCUSSION TITLE
[Esame 31 Agosto 2021] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#247] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-13 194358](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/3eab2cf4-15b3-44fb-945b-e2afd0fe96ab)

**Comment by LeoCosta02** [comment]
Ho pensato di risolvere il punto b con Bernoulli dove con 0 è l evento "pesco dai due mazzi carte dello stesso colore" e con 1 "pesco dai due mazzi carte di colore diverso" la sequenza di "bit" che rappresenta l evento equivale ad avere tutti 1 e la probabilità p è 1/2 (RR NN RN NR) (RN NR) casi favorevoli su casi totali

**Comment by cornflexxx** [comment]
$1)$ 

$$ P(RR) = 20/40 \cdot 20/40 = 1/4$$

$2)$ Fare nessun double rouge $\rightarrow$ ogni  estrazione non è double rouge. $P(RR^c) = 1-P(RR)=3/4$
Analizzando un po' meglio la situazione se otteniamo anche un solo double noir sappiamo con certezza che ci sarà almeno un altro double rouge. Quindi affinché non ci sia nessun double rouge non ci deve essere nessun double noir, avremo quindi solo delle sequenze di estrazioni simili: $\\{R,N\\}$ o $\\{N,R\\}$ , ciascuno dei due casi si ripete 20 volte  disposti in un modo qualunque.

$$P(\text{Nessun double rouge})=(1/2)^{40}$$

$3)$  La distribuzione dei double rouge può essere considerata una binomiale di parametri $n=40,p=1/2$.

**Reply by CasuFrost** [reply]
Riguardo il punto 3, non credo sia giusto in quanto ad ogni estrazione, la probabilità di ottenere un double rouge non è costante, ad esempio, alla prima estrazione è di $1/2\cdot 1/2$, alla seconda estrazione, le probabilità dipendono dalla prima estrazione, ossia 
da quali carte sono rimaste nel mazzo, ad esempio : 
- La prima estrazione ha dato un double rouge : Rimangono 19 rosse nei due mazzi, e le probabilità di un altro RR sono $\frac{19}{39}*\frac{19}{39}$
- La prima estrazione ha dato un double noire : Rimangono 20 rosse nei due mazzi, e le probabilità di un RR sono $\frac{20}{39}*\frac{20}{39}$
- La prima estrazione ha dato una rossa ed una nera, Rimangono 19 rosse nel primo mazzo e 20 rosse nel secondo mazzo,  e le probabilità di un RR sono $\frac{19}{39}*\frac{20}{39}$
- La prima estrazione ha dato una nera ed una rossa, Rimangono 20 rosse nel primo mazzo e 19 rosse nel secondo mazzo,  e le probabilità di un RR sono $\frac{20}{39}*\frac{19}{39}$
\
Quindi credo che non si possa pensare alla distribuzione come una binomiale in quanto la probabilità di ottenere un RR varia.
EDIT : Mi sono reso conto che se con la formula della probabilità totale si calcola quale sia la probabilità di ottenere un RR al secondo lancio, esce come risultato $\frac{1}{4}$, ossia la stessa probabilità di ottenere un RR al primo, che sia $\frac{1}{4}$ la probabilità costante di ottenere un RR? in tal caso è possibile considerare una binomiale di parametro $p=\frac{1}{4}$

**Reply by cornflexxx** [reply]
Sono d'accordo con te, l'unico problema è verificare che questo rapporto rimanga costante e quindi che possa essere approssimato ad una binomiale, ad esempio già verificare che sia valido per la terza estrazione diventa un calcolo abbastanza lungo.

Ho provato diverse volte a simulare le estrazioni delle carte per poi riportarle sul grafico qui sotto:
![doubleRouge](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/121803863/c9984109-8a18-4ab6-83e1-85b51157d0bf)

chiaramente si tratta di una distribuzione binomiale che ho  approssimato ad una distribuzione normale, ricavandomi dalla precedente varianza e media. Nella maggior parte delle simulazioni la media rimane fissa sul $10$ , mentre la varianza si muove  tra $2.5$ e $3$.

Quindi la distribuzione dei double rouge si può approssimare ad una $\mathcal{N} (\mu=10,\sigma=2.5\pm0.5)$

![myplot](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/121803863/afcc19fb-bfba-4a89-8e25-a7e241cbedfb)
